client/src/components/app/AppHeader.js

```diff
@@ -29,6 +32,39 @@ export default function AppHeader() {
           <FaUserAlt className="itemIcon userAvatarIcon" />
         </Link>
       </div>
+
+      <Switch>
+        <Route
+          exact
+          path="/communities/new"
+          render={() => {
+            return (
+              <Modal
+                modalCaption="New Community"
+                crudAction="create"
+                formId={ADD_ACTION_FORM_ID}
+              >
+                <NewCommunityPage formId={ADD_ACTION_FORM_ID} />
```

Comment:
I think formID is not used here

Reply:
True - I put it in use so that I can keep everything DRY and in sync here